marker_size: 0.05
dictionary: "DICT_4X4_50"
